build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+${CARMA_OPT_DIR}/scripts/build_script.sh -p $@ -DBUILD_TESTS=ON -DCREATE_DEB_PACKAGE=ON
```

Comment:
Where is this build_script.sh? Which directory is this env set to ${CARMA_OPT_DIR}?

Reply:
Please see carma-builds